backend/application/syson-sysml-import/src/main/java/org/eclipse/syson/sysml/parser/AstContainmentReferenceParser.java

```diff
@@ -0,0 +1,121 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.syson.sysml.parser;
+
+import java.util.List;
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.syson.sysml.Classifier;
+import org.eclipse.syson.sysml.ConjugatedPortTyping;
+import org.eclipse.syson.sysml.Element;
+import org.eclipse.syson.sysml.Feature;
+import org.eclipse.syson.sysml.FeatureTyping;
+import org.eclipse.syson.sysml.Featuring;
+import org.eclipse.syson.sysml.OwningMembership;
+import org.eclipse.syson.sysml.PortDefinition;
+import org.eclipse.syson.sysml.Redefinition;
+import org.eclipse.syson.sysml.Relationship;
+import org.eclipse.syson.sysml.Specialization;
+import org.eclipse.syson.sysml.Subclassification;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+/**
+ * AstReferenceParser.
+ *
+ * @author gescande.
+ */
+public class AstContainmentReferenceParser {
+
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AstContainmentReferenceParser.class);
+
+    public static void populateContainmentReference(EObject eObject, JsonNode astJson) {
+        List<JsonNode> ownedJson = getOwnedJsonNode(astJson);
+        List<EObject> ownedObject = ownedJson.stream().flatMap(t -> {
+            return AstTreeParser.parseJsonNode(t).stream();
+        }).toList();
+
+        ownedObject.stream().forEach(t -> AstContainmentReferenceParser.ownObject(eObject, t));
+        
+    }
+
+
+    private static List<JsonNode> getOwnedJsonNode(JsonNode astJson) {
+        return ReferenceHelper.extractOwnedObject(astJson);
+    }
+
+    public static void ownObject(EObject owner, EObject owned) {
+        
+        /// Owning Relationship ?s
```

Comment:
Please do not let comments like this one, here and everywhere else in the code.

Reply:
OK